dev/tests/integration/testsuite/Magento/Customer/Model/ResourceModel/AddressRepositoryTest.php

```diff
@@ -328,7 +328,7 @@ public function testSearchAddresses($filters, $filterGroup, $filterOrders, $expe
         }
 
         $searchBuilder->setPageSize(1);
-        $searchBuilder->setCurrentPage(2);
+        $searchBuilder->setCurrentPage(count($expectedResult));
```

Comment:
This change is necessary because some of the tests only expect 1 page back. This change ensures that tests that expect 1 page do not request the second page.